packages/test-exclude/test/test-exclude.js

```diff
@@ -211,10 +223,10 @@ describe('testExclude', () => {
             'coverage/**',
             'packages/*/test/**',
             'test/**',
-            'test{,-*}.js',
-            '**/*{.,-}test.js',
+            'test{,-*}.{,c,m}js',
+            '**/*{.,-}test.{,c,m}js',
             '**/__tests__/**',
-            '**/{ava,babel,jest,nyc,rollup,webpack}.config.js'
+            '**/{ava,babel,jest,nyc,rollup,webpack}.config.{,c,m}js'
```

Comment:
These libraries don't allow config files to have `cjs` or `mjs` extension yet, I made test-exclude support them for future proof.

Reply:
I don't think we should add these options until/unless the projects actually support them, or at least indicate plans to support them.  As an example `ava.config.js` is loaded using the `esm` module, the `.js` extension is explicit in documentation and code.

Reply:
https://github.com/istanbuljs/istanbuljs/pull/381/commits/ed82e4a2af95b9e819437dca5e70ab39a7b6eb0b